release/models/pcep/openconfig-pcep.yang

```diff
@@ -0,0 +1,377 @@
+      	When multiple candidate servers are specified for a single client,
+      	they are prioritized according to preference.";
+    }
+	
+    leaf pce-server-id {
+      type string;
+      description
+	      "A unique name for the pce server.";
+    }
+
+    leaf pce-server-port {
+      type oc-inet:port-number;
+      default 4189;
+      description
+	      "The destination TCP port used for PCEP service in the PCE server.";
+    }
+
+    leaf pce-client-address {
+      type oc-inet:ip-address;
+      description
+	      "The source IP address used by PCC(Router) to establish PCEP session.";
+    }
+	
+    leaf pce-type {
+      type pce-mode-type;
+      description
+	      "The type of PCEP capability supported which is advertised in the Open
+        message when a PCEP session is established.";
+    }
+	
+    leaf preference {
+      type uint8;
+      description
+      	"When multiple PCE servers are specified, the candidate PCE server
+      	with the highest preference is selected to calculate paths.
+      	The greater or lower the value, the higher the preference could
+      	dependon vendor implementation. If no preference value is set, indicate
+      	the lowest preference or no preference at all.";
+    }
+
+    leaf report-local-lsp {
+      type boolean;
+      description
+      	"Specifies whether the PCC(Router) will advertise LSP existence and state
+      	for LSPs that are not controlled by any PCE (for example,LSPs that are
+      	statically configured at the PCC) but without delegating them.";
+     }
+	
+     leaf pce-initiated-capability-enable {
+       type boolean;
+       default false;
+       description
+      	 "Indicates to PCE that PCC(Router) supports PCE-initiated LSP paths instantiation.
+      	 A PCE can initiate LSPs only for PCCs that advertised this capability.";
+     }
+	
+     leaf sr-capability-enable {
+       type boolean;
+       default false;
+       description
+    	  "Indicates to PCE that PCC(Router) supports Segment-Routing
+    	  to set up LSP paths in addition to RSVP-TE.";
+     }
+  }
+
+  grouping pcep-timers-top {
+    description
+      "Configuration and state data for timers applicable
+      to a established PCEP session with pce server.";
+
+    container timers {
+      description
+        "This container defines PCEP timers.";
+
+      container config {
+        description
+          "This container defines PCEP timers configuration.";
+
+        uses pcep-timers-config;
+      }
+
+      container state {
+        config false;
+        description
+          "This container defines state information for PCEP timers.";
+
+        uses pcep-timers-config;
+      }
+    }
+  }
+
+  grouping pcep-timers-config  {
+    description
+      "Defines timers that applies to a PCEP session.";
+
+    leaf keepalive {
+      type uint8;
+      units seconds;
+      default 30;
+      description
+	      "Interval for sending keepalive messages over PCE session.";
```

Comment:
More indentation to fix here and next description

Reply:
Done